docs/configuring-metabase/caching.md

```diff
@@ -103,11 +105,26 @@ To set a caching policy for a question, you must have [curate access](../permiss
 4. Select the [caching invalidation policy](#cache-invalidation-policies).
 5. Save your changes.
 
+## How dashboard, question, database, and default caching policies interact
+
+If you have multiple caching policies set up, Metabase will use the first available policy, in this order:
+
+1. Dashboard
+2. Question
+3. Database
+4. Default (site-wide)
+
+A dashboard policy overrides a question policy, which overrides a database policy, which overrides a default policy.
+
+So if you give a question a caching policy, add that question to a dashboard, then set a caching policy for that dashboard, Metabase would use the dashboard's policy. But that's only when you run the dashboard. If you run the question by itself (outside of the dashboard), Metabase will use the question's policy. If the question appears in multiple dashboards, each with their own policy, the question will use whatever policy is relevant to the current dashboard.
```

Comment:
I got a bit mixed up while reading this and wonder if there's an even simpler way to say it. Hard, though, because the logic just is fairly complicated.

Reply:
We're probably better omitting this explanation entirely, as it's harder to follow than the simple cache hierarchy.